examples/official-storybook/built-storybooks/angular-cli

```diff
@@ -0,0 +1 @@
+../../angular-cli/storybook-static
```

Comment:
How does it work?

Reply:
I just symlinked the built projects into the static directory of this storybook instead of needing to run a HTTP server for those files or importing them with a `file://` (which I guess is conceptually what it does right now).